tutorials/commands/flux-job-cancel.rst

```diff
@@ -0,0 +1,114 @@
+
+    $ flux job cancel ƒh35Dh5qRyq
+
+    <snip wait a little bit>
+
+    $ flux jobs ƒh35Dh5qRyq
+           JOBID USER     NAME       ST NTASKS NNODES     TIME INFO
+     ƒh35Dh5qRyq achu     sleep      CA      1      1   20.18s corona174
+
+In the above example we submitted a simple job via ``flux mini submit`` that simply
+runs ``sleep``.  Passing the resulting jobid to ``flux jobs`` shows that it is
+running (state is ``R``).
+
+We cancel the job simply by passing the jobid to ``flux job cancel``.  After waiting
+a little bit, we see that the job is now canceled in ``flux jobs`` (state is ``CA``).
+
+------------------------
+Cancelling a lot of jobs
+------------------------
+
+The special ``flux job cancelall`` command allows you to cancel many jobs.  Several options allow you to
+target a specific set of jobs.
+
+To start off, lets create 100 jobs that will sleep infinitely.  We will use the special ``--cc`` option
+to ``flux mini submit`` that will submit 100 duplicate copies of the ``sleep`` job.
+
+.. code-block:: console
+
+    $ flux mini submit --cc=1-100 sleep inf
+    <snip - many job ids printed out>
+
+    $ flux jobs
+           JOBID USER     NAME       ST NTASKS NNODES     TIME INFO
+        ƒjTWS5m3 achu     sleep       S      1      -        -
+        ƒjTWS5m4 achu     sleep       S      1      -        -
+        ƒjTWS5m5 achu     sleep       S      1      -        -
+        ƒjTWS5m6 achu     sleep       S      1      -        -
+	<snip - there are many jobs waiting to be run>
+        ƒjTWS5m2 achu     sleep       R      1      1   8.858s corona212
+        ƒjTWS5m1 achu     sleep       R      1      1   8.860s corona212
+        ƒjTUx6Um achu     sleep       R      1      1   8.870s corona212
+        ƒjTUx6Uk achu     sleep       R      1      1   8.870s corona212
+        ƒjTUx6Uj achu     sleep       R      1      1   8.870s corona212
+        ƒjTUx6Ui achu     sleep       R      1      1   8.871s corona212
+	<snip - there are many jobs running>
+
+As you can see, we have a lot of jobs waiting to run (state ``S``) and a lot of running jobs (state ``R``).
+
+Lets first ``flux job cancell`` without any options.
+
+.. code-block:: console
+
+    $ flux job cancelall
+    flux-job: Command matched 100 jobs (-f to confirm)
+
+As you can see, ``flux job cancelall`` found all 100 jobs to cancel, but it hasn't canceled them yet.  In order to go through
+with the cancellation you must specify the ``-f`` option.
+
+.. code-block:: console
+
+    $ flux job cancelall -f
+    flux-job: Canceled 100 jobs (0 errors)
+
+    $ flux jobs
+               JOBID USER     NAME       ST NTASKS NNODES     TIME INFO
+
+As you can see all the jobs are now canceled after passing the ``-f`` option to ``flux job cancelall``.
+
+``flux job cancellall`` has several options to filter the jobs to cancel.  Perhaps the most commonly used
+option is ``-S``.  The ``-S`` option specifies the state of a job to cancel.  The most
```

Comment:
Does this have an equivalent long form? It might be easier to read if it's like `--state` and then the user would remember that (and be able to guess the short form) vs. learning to use `-S` but not knowing what it means.